aiperf/dataset/composer/synthetic.py

```diff
@@ -74,12 +74,12 @@ def _create_turn(self, is_first: bool) -> Turn:
         if self.include_audio:
             turn.audios.append(self._generate_audio_payloads())
 
-        # Add randomized delays between each turn. Skip if first turn.
         if not is_first:
-            turn.delay = utils.sample_positive_normal_integer(
+            delay = utils.sample_positive_normal_integer(
                 self.config.input.conversation.turn.delay.mean,
                 self.config.input.conversation.turn.delay.stddev,
             )
+            turn.delay = delay * self.config.input.conversation.turn.delay.ratio
```

Comment:
So this ratio data member and CLI option was added but just not used before this PR? Was that a mistake from a previous PR?

Reply:
Yes! It looks like a lot of the multi-turn CLI logic up through the dataset was implemented. Since it has not yet been used, any gaps would not have been noticed.